views/js/editProfile.js

```diff
@@ -1,14 +1,14 @@
 function init() {
     var currUser;
     $.ajax({
-        url: 'http://localhost:8080/',
+        url: 'https://localhost:8080/getCurrUser',
```

Comment:
No need to specifically provide IP and port if you provide `/getCurrUser` then also it will work on the base path.